src/DropDownMenu/DropDownMenu.js

```diff
@@ -288,7 +312,7 @@ class DropDownMenu extends Component {
           <div
             style={prepareStyles(Object.assign({}, styles.label, open && styles.labelWhenOpen, labelStyle))}
           >
-            {displayValue}
+            {displayLeftIcon}{displayValue}{displayRightIcon}
```

Comment:
Sorry I wasn't explicit enough with *expose*.
I don't think that we should bake this logic inside the lib. Why? Because the result your are trying to achieve seems pretty specific to your needs. As far as I know, it's not inside the material specifications. 

Could we just expose a fonction so users can do it on their side?